tests/python/pants_test/backend/project_info/tasks/test_export_dep_as_jar.py

```diff
@@ -40,6 +40,10 @@
 from pants.util.dirutil import chmod_plus_x, safe_open
 from pants.util.osutil import get_os_name, normalize_os_name
 
+# runtime_libraries should be the same as compile_libraries for non-strict-deps targets,
+# so we'll often test them together.
+LIBRARIES_ENTRIES = ["compile_libraries", "runtime_libraries"]
```

Comment:
There could also be scoped dependencies in play where compile libs wouldn't be the same as runtime libs, but might be good to call out if it is not in scope.
```
        scoped(
            "some/dep",
            scope = "runtime",
        ),
```

Reply:
I checked that we handle scoped dependencies well, and I think we do: 
```
❯ ./pants export-dep-as-jar testprojects/src/java/org/pantsbuild/testproject/runtime:runtime-pass
{
    "version": "1.1.0",
    "targets": {
        "testprojects/src/java/org/pantsbuild/testproject/runtime:runtime-pass": {
            "targets": [],
            "source_dependencies_in_classpath": [],
            "compile_libraries": [],
            "runtime_libraries": [
                "com.google.code.gson:gson:2.3.1",
                "testprojects.src.java.org.pantsbuild.test.104c409a0cf5bd8e.-d50a15c326d722bebf9123cb8f0c0f1292c7b220"
            ],
            "roots": [
                {
                    "source_root": "/Users/bescobar/workspace/otherpants/testprojects/src/java/org/pantsbuild/testproject/runtime",
                    "package_prefix": "org.pantsbuild.testproject.runtime"
                }
            ],
    ...
```
However, I agree that it's worth updating the comment. Will do, nice catch!